api/src/middleware/downloadFileLocal.ts

```diff
@@ -0,0 +1,35 @@
+import * as fs from 'fs'
+import * as path from 'path'
+
+import type { Middleware } from 'koa'
+import mime from 'mime-types'
+
+import config from '../config'
+import prisma from '../prisma'
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export const downloadFileLocal: Middleware = async function (ctx, next) {
+  // TODO: add authentication
```

Comment:
Hier ist noch eine ToDo

Reply:
Ich hab den Kommentar entfernt und die Authentication bei den beiden Endpunkten nicht weiter umgesetzt.

Diese Endpunkte sind ja nur für die Lokale Entwicklung gedacht, damit man lokal keine Azure-Keys braucht.
In staging oder prod sollten wir sowieso externen blob-provider verwenden.

Daher sehe ich da keine weiteren Sicherheitsbedenken.